frontend/components/flashcard/CardList.tsx

```diff
@@ -40,11 +42,18 @@ export const CardList = ({
 						className="snap-center"
 						key={card.cardID}
 						setCard={setCard}
-						card={{ ...card, deckID: deckID }}
+						card={{ ...card, deckID: deck.deckID }}
 					/>
 				))}
 			</div>
-			<Card card={{ cardID: "", sides: [], deckID: deckID }} />
+			{isAuthorized()?(<Card card={{ cardID: "", sides: [], deckID: deck.deckID }}/>):(<></>)}
```

Comment:
```suggestion
			{ deck.groupRole && GroupRole[deck.groupRole] >= GroupRole.WRITE && <Card card={{ cardID: "", sides: [], deckID: deck.deckID }}/>)}
```
remove function below and insert statement here